platform/configure/config.mdx

```diff
@@ -1,20 +1,24 @@
 ---
-title: vCluster Platform Configuration
-sidebar_label: vCluster Platform Config
+title: Platform Configuration
+sidebar_label: Platform Config
 sidebar_position: 1
+description: Learn how to configure the vCluster Platform, modify its behavior, and enable or disable features such as SSO login, custom branding, and auditing.
 ---
 
 import VClusterProConfig from "../api/_partials/resources/config/status_reference.mdx";
 
-The vCluster Platform config allows you to modify vCluster Platform's behaviour and enable or disable certain features, such as SSO login, custom branding, or auditing.
+Modify the platform's behavior and enable or turn off certain features, such as SSO login, custom branding, or auditing.
 
-## Changing vCluster Platform Config
+## Changing the platform config
 
-vCluster Platform config can be applied directly via helm values or through the vCluster Platform UI in `Admin` > `Config`. Using helm values allows you to declaratively deploy vCluster Platform.
+The platform config can be applied directly via helm values or through the UI in `Admin > Config`. Using helm values allows you to declaratively deploy the platform.
 
-When using helm you can apply the vCluster Platform config via a `vcluster.yaml` file. This also makes it possible to deploy vCluster Platform through GitOps solutions such as ArgoCD. An example vCluster Platform `vcluster.yaml` could look like this:
+When using helm, you can apply the platform config via a
+[vcluster.yaml](/vcluster/configure/vcluster-yaml/external/platform/) file. This
```

Comment:
This is incorrect, The `vcluster.yaml` was incorrectly referenced here. 

This would be `vcluster-platform.yaml` as all these values are for the platform and nothing to do with vcluster itself.